CMakeLists.txt

```diff
@@ -181,7 +181,6 @@ ExternalProject_Add(protobuf
     -DCMAKE_BUILD_TYPE:STRING=RELEASE
     -DBUILD_SHARED_LIBS:STRING=no
     -DCMAKE_INSTALL_PREFIX:PATH=${TRITON_THIRD_PARTY_INSTALL_PREFIX}/protobuf
-  PATCH_COMMAND git cherry-pick -n f180289c4670ca1afde5865bb8a7f2b61a3efcc5
```

Comment:
possible windows build breakage

Reply:
however, the same changes being cherry-picked appear to be on the newly versioned branch v21.12 pulled in by grpc v1.54.3: https://github.com/protocolbuffers/protobuf/blob/f0dc78d7e6e331b8c6bb2d5283e06aa26883ca7c/CMakeLists.txt#L214-L223

(may need to double check versions)